scripts/pr-audit.sh

```
@@ -0,0 +1,509 @@
+sonar_issues=0
+changes_requested=0
+verify_failures=0
+stale_warnings=0
+resolved_count=0
+
+# Current HEAD
+HEAD_SHA=$(git rev-parse --short HEAD 2>/dev/null || echo "unknown")
+HEAD_FULL=$(git rev-parse HEAD 2>/dev/null || echo "unknown")
+
+# Dynamic 24h window
+SINCE=$(date -u -v-24H +"%Y-%m-%dT%H:%M:%SZ" 2>/dev/null || date -u -d "24 hours ago" +"%Y-%m-%dT%H:%M:%SZ" 2>/dev/null || echo "2026-03-14T00:00:00Z")
```

Shellcheck (0.11.0)

[warning] SINCE appears unused. Verify use (or export if used externally).

(SC2034)